payroll-and-expenses/sample-payroll.md

```diff
@@ -0,0 +1,220 @@
+
+![](../assets/in-birthday.png)
+
+### Adding Payroll Column for Employee age
+
+We need to get employee's birthday as a calculation column and write a javascript function to calculate age from that.
+
+![](../assets/in-age.png)
+
+### Adding Payroll Column for Income Tax
+
+We need to use employee's age and basic pay and write a function to calculate income tax.
+
+![](../assets/in-income-tax.png)
+
+### Adding Payroll Column for ESI employee contribution
+
+
+As you have seen you can do some calculations at column level. But things such as tax which are having different percentages and multiple slabs it's better to use saved calculations. Now you should go to **Saved Calculations** tab and add a new saved calculation.
+
+You can create a saved calculation based on a Salary component group \(type\), a Salary component or an Existing payroll column. In this case we use an existing payroll column.
+
+Now click on add button on Calculation Process filed to define the actual calculation. According to our example for the full range of esi is 0.75%. So we don't need to define any ranges and can calculate the tax as follows.
+
+
+
+Now we need to create a payroll column and add this calculation method to that column.
+
+![](../assets/in-esi-employee.png)
+
+Please also play attention how we are assigning **column order** to each column.
+
+
+### Adding Payroll Column for PF employee contribution
+
+![](../assets/in-pf-employee.png)
+
+### Adding Payroll Column for Total Deductions
+
+![](../assets/in-deductions.png)
+
+### Adding Payroll Column for Other Allowances
+
+![](../assets/in-other-allowances.png)
+
+### Adding Payroll Column for ESI employer contribution
+
+![](../assets/in-esi-employer.png)
+
+### Adding Payroll Column for PF employer contribution
+
+![](../assets/in-pf-employer.png)
+
+### Adding Payroll Column for ESI Total
+
+![](../assets/in-esi-total.png)
+
+### Adding Payroll Column for PF Total
+
+![](../assets/in-pf-total.png)
+
+### Add "Net Salary" column
+
+Net Salary is Gross salary minus Total deductions. So we create Net Salary column as shown below:
+
+![](../assets/in-net-salary.png)
+
+Now we have finished defining payroll columns. When you go to Payroll columns and search "IN - " you can see all the fields in Indian payroll.
+
+## Create Payslip Template
+
+As you have all the required payroll columns you can use these to create a payslip template. So we are going to create a new payslip template named Indian Payslip Template and add all the columns defined above. Goto Payslip Templates tab and create a new payslip.
+
+A payslip template has can be created by adding following items:
+
+1. Company Logo
+2. Company Name
+3. Text  \(For adding special messages to notifications to employees\)
+4. Separators \(For separating sections on payslip\)
+5. Payroll columns
+
+
+
+## Create Payroll Report
```

Comment:
Payroll report is included in the payroll import, add a screenshot of this. And mention how to add new columns to payroll report if needed

Following also should be mentioned
- How to change dates of the payroll report (you cant change the dates of an existing report), so need to clone it and change dates

Reply:
can't we change the dates of an existing report?? It can be done